werewolf.rb

```diff
@@ -25,26 +25,23 @@ def send_msg(_to, content)
 player ||= Player.create(phone_number:num)
 session = player.session
 
+if body == "hi"
 
-if body.include? "join"
+send_msg(num,"commands:\nhost\njoin <game_id>\nstatus\ngo\nfinished")
+elsif body.include? "join"
 
 return send_msg(num, "no game id supplied") unless /join (\d|[a-z]){5}/ =~ body
 game_id = body.split(" ")[1]
 s=Session.where(uuid:game_id).first
-if session
-session.players << player
-else
-return send_msg(num, "Could not join game with id of #{game_id}")
-end
-
-if session
-if session.host != player
+if s
+if s.host != player
+s.players << player
 send_msg(num, "Successfully joined game with #{player.session.players.size-1} other players.")
 else
-send_msg(num, "Could not join game with id of #{game_id}")
+send_msg(num, "Already in game with #{player.session.players.size-1} other players.")
```

Comment:
Line is too long. [85/80]<br>Use 2 (not 0) spaces for indentation.<br>Surrounding space missing for operator '-'.